src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -26,12 +27,31 @@ public class Product {
     @JoinColumn(name = "category_id")
     private Category category;
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
could have still used Lombok for generating equals and hashcode and toString, and excluded lazy fields(which would be nice to make category lazy) by annotating them with `@EqualsAndHashCode.Exclude` and the same for toString.
But that is up to you